### DISCUSSION TITLE
is <pthread.h> allowed?

### DISCUSSION THREAD

**Discussion by bssayla** [geeksblabla/blanat#93] [Q&A]
I want to ask if <pthread.h> in c is considered an extern library because I got inspired by NotAsheraf's solution code in Rust but there are no threads in standard libs in c
thanks

**Comment by aboullaite** [comment]
Unfortunately no external libraries are allowed

**Reply by bssayla** [reply]
but isn't "pthread.h" an equivalent of "std::thread" in Rust and "thread" in c++ that already used by other competitors
